pennylane/plugins/default_qubit.py

```diff
@@ -82,6 +84,7 @@
 import itertools
 import warnings
 
+import math
```

Comment:
It looks like you are importing `math` to use `pi`. Since NumPy is already imported, you can remove this and just use `np.pi` below.